packages/eslint-plugin/configs/recommended-with-formatting.js

```diff
@@ -16,6 +16,13 @@ module.exports = {
 		document: true,
 		wp: 'readonly',
 	},
+	settings: {
+		'import/resolver': {
+			node: {
+				extensions: [ '.js', '.jsx' ],
+			},
+		},
+	},
```

Comment:
I think this should be added to the `react` config next to https://github.com/WordPress/gutenberg/blob/eff7ecd690ae3bb1a8579ac0c9152d12d8f7b69e/packages/eslint-plugin/configs/react.js#L4-L6

See also https://github.com/import-js/eslint-plugin-import/blob/master/config/react.js.

Reply:
Good point 👍🏻

Reply:
I updated with https://github.com/WordPress/gutenberg/pull/33746/commits/104fc80490b0d3f62119eee709325447490bdd60, but we would have to retest this change.

@galloppinggryphon, can you check whether adding an override as follows work for you, too:
```js
	settings: {
		'import/extensions': [ '.js', '.jsx' ],
	},
```

It's how they recommend to configure it for JSX.